lib/getstats.js

```diff
@@ -337,72 +337,132 @@ function firefoxGetTrackStats(peerConnection, track, isRemote) {
  * @returns {StandardizedTrackStatsReport}
  */
-function standardizeChromeStats(response, track) {
-  var ssrcReport = response.result().find(function(report) {
-    return report.type === 'ssrc' && report.stat('googTrackId') === track.id;
+function standardizeChromeStats(response) {
+  var inbound = null;
+  var outbound = null;
+  var track = null;
+  var codec = null;
+
+  response.forEach(function(stat) {
+    switch (stat.type) {
+      case 'inbound-rtp':
+        inbound = stat;
+        break;
+      case 'outbound-rtp':
+        outbound = stat;
+        break;
+      case 'track':
+        track = stat;
+        break;
+      case 'codec':
+        codec = stat;
+        break;
+    }
   });
 
+  var isRemote = track && track.remoteSource;
   var standardizedStats = {};
 
-  if (ssrcReport) {
-    standardizedStats.timestamp = Math.round(Number(ssrcReport.timestamp));
-    standardizedStats = ssrcReport.names().reduce(function(stats, name) {
-      switch (name) {
-        case 'googCodecName':
-          stats.codecName = ssrcReport.stat(name);
-          break;
-        case 'googRtt':
-          stats.roundTripTime = Number(ssrcReport.stat(name));
-          break;
-        case 'googJitterReceived':
-          stats.jitter = Number(ssrcReport.stat(name));
-          break;
-        case 'googFrameWidthInput':
-          stats.frameWidthInput = Number(ssrcReport.stat(name));
-          break;
-        case 'googFrameHeightInput':
-          stats.frameHeightInput = Number(ssrcReport.stat(name));
-          break;
-        case 'googFrameWidthSent':
-          stats.frameWidthSent = Number(ssrcReport.stat(name));
-          break;
-        case 'googFrameHeightSent':
-          stats.frameHeightSent = Number(ssrcReport.stat(name));
-          break;
-        case 'googFrameWidthReceived':
-          stats.frameWidthReceived = Number(ssrcReport.stat(name));
-          break;
-        case 'googFrameHeightReceived':
-          stats.frameHeightReceived = Number(ssrcReport.stat(name));
-          break;
-        case 'googFrameRateInput':
-          stats.frameRateInput = Number(ssrcReport.stat(name));
-          break;
-        case 'googFrameRateSent':
-          stats.frameRateSent = Number(ssrcReport.stat(name));
-          break;
-        case 'googFrameRateReceived':
-          stats.frameRateReceived = Number(ssrcReport.stat(name));
-          break;
-        case 'ssrc':
-          stats[name] = ssrcReport.stat(name);
-          break;
-        case 'bytesReceived':
-        case 'bytesSent':
-        case 'packetsLost':
-        case 'packetsReceived':
-        case 'packetsSent':
-        case 'audioInputLevel':
-        case 'audioOutputLevel':
-          stats[name] = Number(ssrcReport.stat(name));
-          break;
-      }
-
-      return stats;
-    }, standardizedStats);
+  function getStatValue(name) {
+    var first = isRemote ? inbound : outbound;
+    var second = track;
+    var third = codec;
```

Comment:
@syerrapragada `first`, `second` and `third` can be declared outside `getStatValue()`.